aws-sdk-s3-transfer-manager/README.md

```diff
@@ -0,0 +1,133 @@
+# AWS SDK S3 Transfer Manager
+
+A high performance Amazon S3 client for Rust.
+
+## :warning: Developer Preview
+
+This library is currently in developer preview and is **NOT** recommended for production environments.
+
+It is meant for early access and feedback purposes at this time. We'd love to hear from you on use cases, feature prioritization, and API feedback.
+
+See the AWS SDK and Tools [maintenance policy](https://docs.aws.amazon.com/sdkref/latest/guide/maint-policy.html#version-life-cycle)
+descriptions for more information.
+
+## Getting started
+
+To begin using the Transfer Manager, follow these examples:
+
+**Add dependency:**
+First, you need to add the dependency in your Cargo.toml file.
+
+```
+aws_sdk_s3_transfer_manager = "0.1.1"
```

Comment:
We can probably drop this instruction about how to add this as (1) crates.io shows how to add this for every crate on the right hand side already and (2) we don't have to keep this version up to date then.